GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -49,12 +49,7 @@ public sealed class RevisionDataGridView : DataGridView
 
         public RevisionDataGridView()
         {
-            _backgroundThread = new Thread(BackgroundThreadEntry)
-            {
-                IsBackground = true,
-                Name = "RevisionDataGridView.backgroundThread"
-            };
-            _backgroundThread.Start();
+            StartBackgroundThread();
```

Comment:
📝 We should eliminate this background thread altogether. Before going into the long-form description, I'm going to see if I can just make this change as a much smaller standalone pull request.

Reply:
This would help with migrartion to .NET Core
https://github.com/gitextensions/gitextensions/pull/7162#discussion_r329353754